### PR TITLE
[AEROGEAR-2639] Add CLI installer script

### PR DESCRIPTION
**Pull request by dimitraz**
**Describe what this PR does and why we need it**:
Create a bash script within the mobile-cli repo to simply and quickly download and install the mobile CLI.

**Progress**
Currently does not work for Windows

**Related Jira**
[AEROGEAR-2639](https://issues.jboss.org/browse/AEROGEAR-2639)

### CODE REVIEW

**Comment by dimitraz** [comment]
ping @maleck13 @PhilipGough @JameelB @philbrookes

**Comment by maleck13** [comment]
This looks ok to me. Can we add instructions to the readme

File: README.md [hunk 1]
```diff
@@ -23,7 +23,18 @@ mobile get clientconfig <mobileClientID> --namespace=<namespace>
 mobile create integration <consumingServiceInstanceID> <providingServiceInstanceID> --namespace=<namespace>
 ``` 
 
-## CLI Installation
+## Installing the CLI
+To quickly get up and running with the CLI for Mac and Linux, simply executre the installer script from inside the `scripts` directory.
```
Comment (JameelB) [inline README.md]: typo `execute`

**Comment by maleck13** [comment]
merging as only doc and unreleated script change

**Pull request merged**